Libraries/Image/RCTImageUtils.m

```diff
@@ -314,14 +330,18 @@ BOOL RCTUpscalingRequired(CGSize sourceSize, CGFloat sourceScale,
   return (__bridge_transfer id)imageProperties;
 }
 
-NSData *__nullable RCTGetImageData(CGImageRef image, float quality, NSInteger orientation)
+NSData *__nullable RCTGetImageData(CGImageRef image, float quality, UIImageOrientation imageOrientation)
```

Comment:
What a second, what if we make `RCTGetImageData` accept `UIImage` instance instead of `CGImage` and `UIImageOrientation`? Looks like it will be much simpler!